src/components/footer.js

```diff
@@ -51,48 +51,23 @@ function Footer() {
           <Logo
             size="medium"
             image={
-              <img
-                
-                alt={t("footer.secondary.heading.logo")}
-                src={logoImg}
-              />
+              <img alt={t("footer.secondary.heading.logo")} src={logoImg} className="footer__logo" />
```

Comment:
Can you think of a better way to arrange the order of attributes?

Reply:
What do you think? Alphabetical?

Reply:
Order of importance: className, src, alt

Reply:
Okay I put class name first in all the components, except for links, which have `to` and sometimes one other attribute (like `onClick`) first